__tests__/app.test.js

```diff
@@ -0,0 +1,65 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const {
+  topicData,
+  userData,
+  commentData,
+  articleData,
+} = require("../db/data/test-data/index");
+
+beforeAll(() => {
+  return seed({
+    topicData,
+    userData,
+    commentData,
+    articleData,
+  });
+});
+
+afterAll(() => db.end());
+
+describe("App test", () => {
+  describe("GET /api/topics", () => {
+    test("200 :returns topic", () => {
+      const obj = [
+        { slug: "mitch", description: "The man, the Mitch, the legend" },
+        { slug: "cats", description: "Not dogs" },
+        { slug: "paper", description: "what books are made of" },
+      ];
+      return request(app)
+        .get("/api/topics")
+        .send(obj)
+        .expect(200)
+        .then((res) => {
+          console.log(res.body);
+          expect(res.body).toEqual({ topic: obj });
+        });
+    });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("200 :returns article by id", () => {
+    const id = [
+      {
+        article_id: 12,
+        title: "Moustache",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "Have you seen the size of that thing?",
+        created_at: "2020-10-11T11:24:00.000Z",
+        votes: 0,
+      },
+    ];
+    return request(app)
+      .get("/api/articles/12")
+      .send(id)
+      .expect(200)
+      .then((res) => {
+        console.log(res.body);
+        expect(res.body).toEqual({ topic: id });
+      });
+  });
+});
```

Comment:
Also need to add some error handling here 
- what if we had an id of 26354
- what if we had an id of this-is-a-string